src/main/java/com/atlassian/bitbucket/jenkins/internal/link/BitbucketLinkType.java

```diff
@@ -0,0 +1,17 @@
+package com.atlassian.bitbucket.jenkins.internal.link;
+
+public enum BitbucketLinkType {
+
+    BRANCH("View Branch"),
+    REPO("Browse Repo");
```

Comment:
May I propose we don't shorten this? so `Browse Repository` 

Also, these need to be i18n'ed right? So while this is simple here, I think we need to do something to look up the i18n strings. Either here or when we call the `getDisplayName`